allen-cell-animated/cell-catalog
# add protocols data

Problem
=======
What is the problem this work solves, including
closes #56 

Solution
========
What I/we did to solve this problem
added fields for protocols data to config file 
with @meganrm @interim17 @tyler-foster 

text and format reviewed by @TravisKroeker 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. `npm run`
2. `npx netlify-cms-proxy-server`
3.  check http://localhost:8000/admin/#/collections/disease-cell-line/entries/AICS-104/index

Screenshots (optional):
-----------------------
<img width="666" alt="Screenshot 2024-09-12 at 2 26 07 PM" src="https://github.com/user-attachments/assets/ae637975-93a0-414c-8631-da808af685bd">
